packages/contracts-bedrock/scripts/Artifacts.s.sol

```diff
@@ -81,6 +85,28 @@ abstract contract Artifacts {
         }
     }
 
+    function hasSuffix(string memory _fullString, string memory _suffix) public pure returns (bool) {
```

Comment:
We should see if solady has a helper here for this so we don't need to implement/test our own code

Reply:
I am splitting this PR so that we can go ahead and update op-node now. I will update the contracts deploy script as part of https://github.com/ethereum-optimism/optimism/pull/11175 instead